lib/config/presets/internal/group.ts

```diff
@@ -484,6 +485,30 @@ const staticGroups = {
       },
     ],
   },
+  rubyOnRails: {
+    description: 'Group Ruby on Rails packages together.',
+    packageRules: [
+      {
+        matchDatasources: ['rubygems'],
+        matchPackageNames: [
+          'actioncable',
+          'actionmailbox',
+          'actionmailer',
+          'actionpack',
+          'actiontext',
+          'actionview',
+          'activejob',
+          'activemodel',
+          'activerecord',
+          'activestorage',
+          'activesupport',
+          'railties',
+          'rails',
```

Comment:
Should we group all `rails` dependencies?

You can use _some_ dependencies independently outside of Rails. The Ruby on Rails readme says: [^readme]

> [Active Record](https://github.com/rails/rails/blob/main/activerecord/README.rdoc), [Active Model](https://github.com/rails/rails/blob/main/activemodel/README.rdoc), [Action Pack](https://github.com/rails/rails/blob/main/actionpack/README.rdoc), and [Action View](https://github.com/rails/rails/blob/main/actionview/README.rdoc) can each be used independently outside Rails.

[^readme]: [Rails readme, frameworks and libraries](https://github.com/rails/rails#frameworks-and-libraries)

Reply:
they can still be used as is, even if they are grouped

the group is only to group all available updates together, so if you only use the `active...` you only get those in one pr